src/pages/tokenDrop/TokenDrop.tsx

```diff
@@ -35,12 +37,14 @@ const TokenDrop: FunctionComponent = () => {
       <Box
         sx={ {
           display: ["none", "none", "flex"],
-          position: "absolute",
+          position: "fixed",
           top: 32,
           right: [0, 0, 32, 160],
         } }
       >
-        <NEWMLogo width={ 110 } height={ 110 } />
+        <a href="https://www.newm.io" target="_blank" rel="noopener noreferrer">
```

Comment:
Last time i talked to Tizi she didnt want the icon to fwd them to our newm page because it was not consistent throughout the steps and it would take them away from the flow